pom.xml

```diff
@@ -219,7 +223,7 @@
         <version.org.wildfly.openssl.wildfly-openssl-solaris-x86_64>${version.org.wildfly.openssl.natives}</version.org.wildfly.openssl.wildfly-openssl-solaris-x86_64>
         <version.org.wildfly.openssl.wildfly-openssl-windows-i386>${version.org.wildfly.openssl.natives}</version.org.wildfly.openssl.wildfly-openssl-windows-i386>
         <version.org.wildfly.openssl.wildfly-openssl-windows-x86_64>${version.org.wildfly.openssl.natives}</version.org.wildfly.openssl.wildfly-openssl-windows-x86_64>
-        <version.org.wildfly.security.elytron>1.11.0.CR1</version.org.wildfly.security.elytron>
+        <version.org.wildfly.security.elytron>1.11.0.CR2-SNAPSHOT</version.org.wildfly.security.elytron>
```

Comment:
This change is causing CI tests to fail as artifact cannot be found.

Reply:
Sorry about that, that snuck in by mistake. CI will still fail though as this PR depends on the corresponding Elytron PR.